VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -275,13 +275,12 @@ public void onSuccess(AuthResponse result) {
                             }
                         } catch (LoginException ex) {
                             super.onException(ex);
+                            onUserLoginFailure(ex, null, null);
                         }
                     }
-
                     @Override
                     public void onException(Exception ex) {
                         super.onException(ex);
-                        onUserLoginFailure(ex, null, null);
```

Comment:
Yeah, you can simply call the `tryToSetUIInteraction(true);` here I guess, to avoid the freezing issue that's being fixed here. The standard error message handling should already be done by the `super.onException(ex);` call.